soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3975,6 +3976,19 @@ void CreateScrubMessages() {
     }
 }
 
+CustomMessageMinimal NaviMessages[];
```

Comment:
You should either remove this here, or Remove the `CustomMessageMinimal` type declaration part from line 4083 and specify the size of the array up here instead of down there. The two errors you are getting on Linux are: `array size missing in 'NaviMessages'` on this line and `conflicting declaration 'CustomMesasgeMinimal NaviMessages [15]'` on line 4038.

Additionally, you should probably `#define NUM_NAVI_MESSAGES 15` or something somewhere (maybe `randomizerTypes.h`?) and then use that value when declaring the array size. That way if someone in the future wants to add some there's an easier location to find where to increase the size of the array.

Reply:
Thanks @leggettc18!  Yep, that seems to have fixed it.  I had to reorder the code a bit to put the text array above `CreateNaviRandoMessages()` to avoid the redundant declaration (I originally had it that way because I was trying to put the big block of text at the very bottom for aesthetic purposes).  For some reason VS didn't like having it down below without it being defined a second time.

Anyways, Jenkins is happy now, and I verified that all 3 languages are still working.